app/components/step-footer.jsx

```diff
@@ -60,6 +62,13 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
   },
   next: {
     margin: '1rem 2rem',
+    width: '100%',                 // Flexible width based on parent
+    maxWidth: '13.8125rem',        // Ensures it doesn't exceed the design width
+    height: '3.125rem',
+    padding: '0.5rem 1.25rem',
+    borderRadius: '0.5rem',
+    border: `2px solid ${theme.colorPrimary}`,
```

Comment:
Okay colorPrimary does not exist in theme.js and it should be theme.colors.something  I know you didn't write this originally, but can you find the right color in theme.js and use it, both here and elsewhere.